spec/descriptions/patchSyntheticTest.md

```diff
@@ -1,16 +1,22 @@
 This API endpoint updates selected attributes of a Synthetic Test.
 
 - All attributes listed as in the schema, including the required ones, are optional for this call.
+- Synthetic Test configuration properties set to null will be removed from the configuration.
+- For major updates to the Synthetics Test or to remove main attributes, see "Update a Synthetic test"
 
 ## Sample script and payload: 
-- A sample script to patch a simple Ping API test to disable it.
+- A sample script to patch a simple HTTP Script Test to enable it and to switch from multi-scripts to single script.
 
 ```
 curl -k -v -X PATCH \
 https://<Host>/api/synthetics/settings/tests/Ilfs9bW97KkTxuyGtxBF \
 -H 'authorization: apiToken <Token>' \
 -H 'content-type: application/json' \
 -d '{
-    "active":false
+    "active" : enable,
```

Comment:
If the default value of active is changed from true/false to enable/disable, then this line should be updated: 
https://github.com/instana/openapi/blob/5853134014b82c8b582d92f76f7b7f6bf3528a81/spec/descriptions/tagSyntheticSettings.md?plain=1#L26

Reply:
no, I did not change the type... it is still a boolean. I will fix the sample and change enable to `true`